flink-core/src/main/java/org/apache/flink/configuration/RestartBackoffTimeStrategyOptions.java

```diff
@@ -46,7 +57,7 @@
 		.withDescription("Time interval in milliseconds for measuring failure rate.");
 
 	/**
-	 * Backoff time between two consecutive restart attempts in FailureRateRestartBackoffTimeStrategy.
+	 * Backoff time (milli-seconds) between two consecutive restart attempts in FailureRateRestartBackoffTimeStrategy.
 	 */
 	@PublicEvolving
 	public static final ConfigOption<Long> RESTART_BACKOFF_TIME_STRATEGY_FAILURE_RATE_FAILURE_RATE_BACKOFF_TIME = ConfigOptions
```

Comment:
I know this discussion is out of scope for this PR but do we need new config keys? For backwards compatibility we should respect the existing [config keys](https://github.com/apache/flink/blob/0a405251b297109fde1f9a155eff14be4d943887/flink-core/src/main/java/org/apache/flink/configuration/RestartStrategyOptions.java) anyways.

Reply:
Good question. Previously the configs for strategy strategies are quite a mess in ConfigConstants and I wanted to deprecate them. But with FLIP 61 the configs are better organized in RestartStrategyOptions.java.

I think now we can use the configs from RestartStrategyOptions.java directly. And I'm also thinking to rename RestartBackoffTimeStrategy to RestartStrategy once we have removed the legacy RestartStrategy.

I will open a hotfix commit to migrate these options and their usages in related strategy.